src/session.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  bufferToFile,
+  fileToBuffer,
+  mySessionItems,
+  mySessionMeat,
+  toItem,
+} from "kolmafia";
+import { getFoldGroup } from "./lib";
+import { $item, $items } from "./template-string";
+import { sumNumbers } from "./utils";
+
+/**
+ * Return a mapping of the session items, mapping foldable items to a single of their forms
+ * @returns the item session results, with foldables mapped to a single of their folding forms
+ */
+function mySessionItemsWrapper(): Map<Item, number> {
+  const manyToOne = (primary: Item, mapped: Item[]): [Item, Item][] =>
```

Comment:
I kinda of want to yoink these `const`s out--it feels like they could be applicable outside of this function